packages/flutter_tools/lib/src/ios/code_signing.dart

```diff
@@ -8,14 +8,22 @@ import 'package:process/process.dart';
 
 import '../base/common.dart';
 import '../base/config.dart';
+import '../base/file_system.dart';
 import '../base/io.dart';
 import '../base/logger.dart';
 import '../base/platform.dart';
 import '../base/process.dart';
 import '../base/terminal.dart';
 import '../convert.dart' show utf8;
+import 'plist_parser.dart';
 
 const String _developmentTeamBuildSettingName = 'DEVELOPMENT_TEAM';
+const String _codeSignStyleBuildSettingNAme = 'CODE_SIGN_STYLE';
+const String _provisioningProfileSpecifierBuildSettingName = 'PROVISIONING_PROFILE_SPECIFIER';
+const String _provisioningProfileBuildSettingName = 'PROVISIONING_PROFILE';
+
+const String _codeSignSelectionCanceled =
+    'Code-signing setup canceled. Your changes have not been saved.';
```

Comment:
Nit: add `k` prefix for global constants: https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#begin-global-constant-names-with-prefix-k

Reply:
Mmm I don't know if these are considered global constants since they're private?

Reply:
It looks like we use this style for private globals too, see: https://github.com/search?q=repo%3Aflutter%2Fflutter+lang%3Adart+%22const+String+_k%22&type=code

I don't feel strongly about this though since this passes the formatter.